src/structures/MessageButton.js

```diff
@@ -163,3 +163,8 @@ class MessageButton extends BaseMessageComponent {
 }
 
 module.exports = MessageButton;
+
+/**
+ * @external APIButtonComponent
```

Comment:
To be consistent with the names we use on the library
```suggestion
 * @external APIMessageButton
```

Reply:
I chose this because discord-api-types uses the same name, and provides the same data. You sure you want to use this name still?

Reply:
Hold on a second... this already exists:
https://github.com/discordjs/discord.js/blob/d15dd5f07dab00e8a31f0a37b1e60ea4017871d0/src/structures/MessageComponentInteraction.js#L112-L120

Reply:
I'll use those for now